charts/opentelemetry-collector/values.yaml

```diff
@@ -180,6 +180,12 @@ ports:
     servicePort: 9411
     hostPort: 9411
     protocol: TCP
+  metrics:
+    enabled: true
+    containerPort: 8888
+    servicePort: 8888
+    hostPort: 8888
```

Comment:
Suppose we don't need the `hostPort` for metrics.

Reply:
Agreed, added it because of consistency but probably better to remove it.